_posts/2023-01-01-seraphis-jamtis-developer-opportunities.md

```diff
@@ -0,0 +1,55 @@
+
+Work has now started on the so-called *Seraphis wallet* that will make heavy use of the Seraphis library. *wallet* in this context does not mean an app that end users run to manage their XMR, but that component of the Monero core code that such apps will build on. The main functionality of that component: Reading and writing wallet files that contain keys and all owned outputs, scanning the blockchain for new incoming transactions, building new transactions and submitting them to the network.
+
+The current plan is to fully replace the current component with this role called *wallet2*. This in turn will make many adjustments in the rest of the Monero core codebase necessary, and of course reworking all wallet apps to use the new component. You find more details [here](https://github.com/seraphis-migration/strategy/wiki).
+
+### Financial Matters
+
+There is no company behind the Monero Project, it has no employees: Devs seeking more or less conventional employment best look elsewhere. It does not actively seek venture capital either. Monero runs on contributions from people that basically work "for free", and on donations from the broader community to finance larger tasks.
+
+One of these funding methods is the *CCS*, the *Community Crowdfunding System*. You can find more details [on the website](https://ccs.getmonero.org/).
+
+Basically anybody can write a proposal for it, and if it gets positive feedback it moves to the funding stage where of course success depends on whether donators will find the proposed work and goal worthwhile.
+
+Realistically however it's pretty hard to get a proposal accepted if you are all new to Monero, nobody in the community knows you yet and you have never contributed anything so far. Much is about trust and demonstrated abilities, as it typically is the case in open source projects.
+
+The CCS has a long history of successfully funded and executed tasks.
+
+Another option for funding Monero related work is the [MAGIC Monero Fund](https://magicgrants.org/funds/monero). Its nature is quite different from the CCS as it's a USA-registered 501(c)(3) charity. This enables tax-deductible (for citizens of the USA) donations in a variety of methods, and the option of USD-denominated payouts. There is also an additional requirement that fund recipients must verify their identity. The CCS allows its fund recipients to remain anonymous, yet only handles XMR.
+
+### Conclusion
+
+There will be a lot of interesting work for devs that want to join the Monero project, for a long time. It may all look a bit unusual as a "place" to work at, and "get rich quick" isn't, but on the other hand think about it: Where else can you make a difference for a multi-billion-dollar venture that may become important in the future and make a difference, whoever you are, from wherever you are, regardless of education, only based on your knowledge and your willingness to make useful contributions?
```

Comment:
> and "get rich quick" isn't 

hm.. if - "but it isn't" is used: "but on the other hand" would have to be changed.

Reply:
Because of your comment I noticed that "get rich quick" is the wrong metaphor to use because that hints at scams or at least shady things. That's not what I want to say here however, thus I arrived at the following proposal:

> It may be a bit unusual as a "place" to work at, and for getting rich probably look elsewhere, but on the other hand think about it: